.gitmodules

```diff
@@ -19,6 +19,6 @@
 [submodule "subprojects/h2o"]
 	path = subprojects/libh2o
 	url = https://github.com/urbit/h2o.git
-[submodule "subprojects/argon2"]
-	path = subprojects/argon2
-	url = https://github.com/urbit/argon2.git
+[submodule "subprojects/secp256k1"]
+	path = subprojects/secp256k1
+	url = https://github.com/urbit/secp256k1.git
```

Comment:
This looks like the same bad conflict resolution as in `meson.build` -- both submodules should be present.